pkg/spdk_test.go

```
@@ -144,6 +144,8 @@ func LaunchTestSPDKGRPCServer(ctx context.Context, c *C, ip string, execute func
 	go func() {
 		<-ctx.Done()
 		spdkGRPCServer.Stop()
+		logrus.Info("Stopping SPDK gRPC server")
+		util.ForceStopSPDKTgtDaemon(5 * time.Second)
```

golangci-lint (1.62.2)

Error return value of `util.ForceStopSPDKTgtDaemon` is not checked

(errcheck)